### PR TITLE
Add logstash_transit container

### PR DESCRIPTION
**Pull request by kenoir**

### CODE REVIEW

File: logstash_transit/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM docker.elastic.co/logstash/logstash:7.2.0
+
+LABEL maintainer "Robert Kenny <R.Kenny@wellcome.ac.uk>"
```
Comment (jamesgorrie) [inline logstash_transit/Dockerfile]: Worth using a more generic email?

File: logstash_transit/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+LABEL description "A Docker image for deploying logstash"
+
+USER 0
+
+# Installs envsubst on Centos
+RUN yum install -y gettext
+
+USER 1000
```
Comment (jamesgorrie) [inline logstash_transit/Dockerfile]: What's this?

File: logstash_transit/pipeline/udp.conf.template [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+output {
+  elasticsearch {
+    hosts    => "${ES_HOST}"
+    user     => "${ES_USER}"
+    password => "${ES_PASS}"
+    ssl      => true
+  }
+
```
Comment (jamesgorrie) [inline logstash_transit/pipeline/udp.conf.template]: You can add a property here:

`add_field => {"service_name" => "$SERVICE_NAME"}` - but then you would have to make sure that the service has this set.

Comment (jamesgorrie) [reply logstash_transit/pipeline/udp.conf.template]: actually this won't work as we are having 1 log service per cluster, so this needs to be dynamic.

File: logstash_transit/Dockerfile (hunk 2 above)
Comment (kenoir) [reply logstash_transit/Dockerfile]: That's the original user ID the parent dockerfile sets - TLDR: I don't know.

File: logstash_transit/Dockerfile (hunk 1 above)
Comment (kenoir) [reply logstash_transit/Dockerfile]: Yes!

**Pull request merged**